browser/src/control/Control.JSDialog.js

```diff
@@ -700,7 +700,7 @@ L.Control.JSDialog = L.Control.extend({
 			instance.updatePos = this.setPosition.bind(this, instance);
 
 			// Special case for nonModal dialogues. Core side doesn't send their initial coordinates. We need to center them.
-			if (instance.nonModal) {
+			if (instance.nonModal && !(instance.startX && instance.startY)) {
```

Comment:
why it is needed? how to test?
best to do such modifications of common code in separate commit with description

Reply:
We have tree type of dialog:
    * Modal (isModalPopUp = true): non-movable + overlay + dimmed background.
    * Nonmodal: movable + no dim + no overlay (user can interact with the document).
    * Popup (Non-dialog) (isDocumentAreaPopup = true): overlay + no dim.
But i need a new type: no dim + no overlay + no titlebar(same like popup)

"Overlay" prevents  selecting a cell, Formula usage dialog has position info but not popup. Popups covers a layer and disable to selecting a cell. So I've solved the positioning problem like that.

Reply:
so can we explicitly define such type and name it? so we will have it documented in the comment and `instance.isYourNewType` property set ?

Reply:
I'm not sure that is a new type just a nonmodal dialog without title but have position info

Reply:
But maybe we can use isFormulaUsage same like isAutofilter?

Reply:
I prefer some generic name, for a "popup without overlay" - so we can later reuse the same thing without confusion ion other cases.